pkg/api/config/config.go

```diff
@@ -101,7 +101,13 @@ type SchedulerConfig struct {
 	NumWorkers int
 }
 
+type LDAPCredentials struct {
+	BindDN       string
+	BindPassword string
+}
+
 type LDAPConfig struct {
+	CredentialsFile    string
 	Port               int
```

Comment:
BindPassword string below should be removed right? In fact, isn't that the only credential?

Reply:
That is the password, the username is the other property.

And why remove it from here? I thought this feature was optional?

Reply:
Optional in the sense the user may choose to keep his credentials in the config, or he may choose to save them in the credentials file.

Reply:
No, we always want separate credentials file